08-communicating-results.Rmd

```diff
@@ -261,7 +263,7 @@ Reproducibility requires that we consider several key components:
 
 Many tools, practices, and project management techniques exist to make survey analysis projects easy to reproduce. For best results, they should be decided upon and applied at the beginning of a project. Below are our suggestions for a survey analysis data workflow. This list is not comprehensive but aims to provide a starting off point for teams looking to create a reproducible workflow.
 
-### Setting Random Number Seeds
+### Setting Random Number Seeds  
```

Comment:
How often are seeds used in analysis? This is more in sampling or simulation.

Reply:
I had it in my notes to add `set.seed()` in reproducibility, but we can move it elsewhere/remove if we don't use it anywhere.